### PR TITLE
Adjust readmes

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Refactor README to be more readable.

### CODE REVIEW

File: notebooks/thoth-performance-dataset/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Thoth Performance Datasets
+
+Thoth Performance Datasets are created with one of the components of Thoth called [Amun](https://github.com/thoth-station/amun-api).
+This service acts as an execution engine for Thoth where applications are built and tested using [Thoth Performance Indicators (PI)](https://github.com/thoth-station/performance).
+Amun can be scheduled through another component in Thoth called [Dependency Monkey](https://github.com/thoth-station/adviser/blob/master/docs/source/dependency_monkey.rst).
```
Comment (harshad16) [inline notebooks/thoth-performance-dataset/README.md:5]:
```suggestion
Amun can be scheduled through another component in Thoth called [Amun API](https://github.com/thoth-station/amun-api/blob/master/README.rst).
```
Shouldn't this be amun-api?

Comment (pacospace) [reply notebooks/thoth-performance-dataset/README.md:5]: It's correct actually, a little bit above I mention Amun 

```
Thoth Performance Datasets are created with one of the components of Thoth called [Amun](https://github.com/thoth-station/amun-api).
```

then I explain that we can use DM to schedule inspections through Amun. 

But I see the point, I will be more verbose to make it more clear. Thanks for pointing that out :) :100:

Comment (harshad16) [reply notebooks/thoth-performance-dataset/README.md:5]: oh ok, thanks for the explanation.

**Review by fridex — APPROVED** [review]

**Pull request merged**